libraries/libfc/include/fc/io/datastream.hpp

```diff
@@ -295,6 +296,35 @@ inline datastream<ST>& operator>>(datastream<ST>& ds, uint8_t& d) {
   ds.read((char*)&d, sizeof(d) );
   return ds;
 }
+
+// write to stream
+template<typename ST, typename T>
+inline datastream<ST>& operator<<(datastream<ST>& ds, const boost::dynamic_bitset<T>& bs) {
+  std::vector<uint32_t> blocks;
+  blocks.resize(bs.num_blocks());
+  boost::to_block_range(bs, blocks.begin());
+
+  for (const auto& b: blocks) {
+    ds.write( (const char*)&b, sizeof(b) );
+  }
+
+  return ds;
+}
+
+// read from stream
+template<typename ST, typename T>
+inline datastream<ST>& operator>>(datastream<ST>& ds, boost::dynamic_bitset<T>& bs) {
+  std::vector<uint32_t> blocks;
+  blocks.resize(bs.num_blocks());
+
+  for (auto& b: blocks) {
+    ds.read( (char*)&b, sizeof(b) );
+  }
+
+  bs = { blocks.cbegin(), blocks.cend() };
+  return ds;
+}
```

Comment:
This should not be here. Why is it needed at all? Seems we should add to/from `variant` and/or `pack/unpack` and not `operator<</>>`

Reply:
Can you push up the changes. I just tried and it worked for me.

Reply:
Your current build issue is because you don't declare or define `pack/unpack` for `dynamic_bitset` above the default. Forward declare them like `multiprecision::number`.

Reply:
Thanks! That worked; I have added tests for packing/unpacking too. Will incorporate your other comments in next commits.